python/cudf/cudf/core/column/column.py

```diff
@@ -179,32 +179,24 @@ def equals(self, other: ColumnBase, check_dtypes: bool = False) -> bool:
     def all(self, skipna: bool = True) -> bool:
         # If all entries are null the result is True, including when the column
         # is empty.
-        result_col = self.nans_to_nulls() if skipna else self
```

Comment:
Why this change?

Reply:
I'm confused by the same below?

Reply:
Oh I see this method is empty and has just been removed as we discussed and only is defined for numerical now. OK we're good.